android/libraries/rib-base/build.gradle

```diff
@@ -43,7 +43,7 @@ dependencies {
     implementation deps.apt.javaxInject
 
     implementation deps.uber.autodisposeCoroutines
-    implementation deps.kotlin.coroutinesAndroid
+    api deps.kotlin.coroutines
```

Comment:
This needs API now because we are exposing (`internal`) a `lifecycleFlow` which is a `SharedFlow`, so extenders must know `SharedFlow` at compile time (even though they can't access it).

Also, changed from `coroutinesAndroid` to `coroutines` -- this is not even an Android package, why are we using `coroutinesAndroid`?

Reply:
Making the property `@JvmSynthetic` solves this issue -- reverted to `implementation`, but kept the change from `coroutinesAndroid` to `coroutines`